desktop/libs/hadoop/src/hadoop/core_site_tests.py

```diff
@@ -97,7 +96,6 @@ def test_core_site():
     assert_equal(core_site.get_default_fs(), 'abfs://data@gethuedevstorage.dfs.core.windows.net/hue-adls')
 
     assert_equal(get_remote_home_storage(), 's3a://gethue-dev')
-    assert_equal(get_region(), 'us-west-2')
```

Comment:
Kind of scary when a change has to modify the existing unit tests.. could there be real scenarios that depend on this behaviour?

Reply:
No we are completely removing that scenario of depending on core-site for bucket region because of flakiness.
From now, we only depend on Hue config for it.